firmware/serial_debugger/debug_modules.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+
+message GPIO {
+  uint32 pin = 1;
+  bool value = 2;
+}
+
+message ADC_Channel {
+  float value = 1;
+}
```

Comment:
nitpick: We use `PascalCase` for type names, i.e. `Gpio` and `AdcChannel`
Maybe also distinguish between `GpioWrite` and `GpioRead`, because eventually they're gonna be different, but fine for now since this is really just settting up the build system

Reply:
will add gpio write and read when adding the enums in next PR